Gordon360/Controllers/RecIM/MatchesController.cs

```diff
@@ -80,6 +85,62 @@ public ActionResult<IEnumerable<LookupViewModel>> GetMatchTypes(string type)
             return NotFound();
         }
 
+        /// <summary>
+        /// Gets all surfaces
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("surfaces")]
+        public ActionResult<IEnumerable<SurfaceViewModel>> GetSurfaces()
+        {
+            return Ok(_matchService.GetSurfaces());
+        }
+
+        /// <summary>
+        /// Creates a new match/series surface
+        /// </summary>
+        /// <param name="newSurface"></param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("surfaces")]
+        [StateYourBusiness(operation = Operation.ADD, resource = Resource.RECIM_MATCH)]
```

Comment:
Technically since this resource is nested inside the Match Controller, the Match is still the resource. If I made a second controller base for surfaces, I would definitely need a new StateYourBusiness resource

Reply:
For the record, Authorization resources do not map 1:1 with REST resources - the word is overloaded but they are not the same thing.

You definitely do need a different resource if the authorization rules should be different between the two resources. If the rules are the same, it's a little bit less obvious when something should be treated as a separate resource for authorization.

Reply:
ill make a seperate resource